src/Package/Impl/DataInspect/VariableGridHost.xaml.cs

```diff
@@ -54,7 +54,7 @@ public partial class VariableGridHost : UserControl {
         internal void SetEvaluation(VariableViewModel wrapper) {
             VsAppShell.Current.AssertIsOnMainThread();
 
-            if (wrapper.TypeName == "NULL" && wrapper.Value == "NULL") {
+            if (wrapper.TypeName == null && wrapper.Value == null) {
```

Comment:
Perhaps this should be an OR? Will the other branch actually work if one is null, but the other one isn't?